src/ceph-disk

```diff
@@ -1593,7 +1603,10 @@ def prepare_dev(
         except subprocess.CalledProcessError as e:
             raise Error(e)
 
-        rawdev = get_partition_dev(data, 1)
```

Comment:
get_partition_dev_by_partuuid(osd_uuid)

needs to wait for the udev event that will create the partuuid symlink to run. The

    get_partition_dev(data, 1)

call does not need to wait because it does not rely on something that happens in the background